README.md

```diff
@@ -26,3 +26,4 @@ TBD
 |lenskit(rec system)  |https://lenskit.org/  | https://github.com/lenskit/lkpy
 |Submitty  | https://submitty.org/       | https://submitty.org/developer/how_to_contribute
 |NumPy  |https://numpy.org/     | https://docs.scipy.org/doc/numpy/dev/
+|Theano |https://github.com/Theano/Theano |http://www.deeplearning.net/software/theano/
```

Comment:
Looks good, can you add http://deeplearning.net/software/theano/dev_start_guide.html link for contributing guide.